app/src/main/java/org/fossasia/openevent/app/data/models/Attendee.java

```diff
@@ -93,34 +94,7 @@ public Attendee withCheckedIn() {
 
     @Override
     public int compareTo(@NonNull Attendee other) {
-        int compareFirstName;
-        int compareLastName;
-
-        // TODO: Reduce complexity and move logic outside the class
-        return
-            (compareFirstName = firstName
-                .toLowerCase()
-                .compareTo(
-                    other
-                        .getFirstName()
-                        .toLowerCase()
-                )
-            ) == 0 ? (compareLastName =
-                lastName
-                    .toLowerCase()
-                    .compareTo(
-                        other
-                            .getLastName()
-                            .toLowerCase()
-                    )
-            ) == 0 ?
-                email
-                    .toLowerCase()
-                    .compareTo(
-                        other
-                            .getEmail()
-                            .toLowerCase()
-                    ) : compareLastName : compareFirstName;
+        return getFirstName().compareTo(other.getFirstName());
```

Comment:
Will add lowercase but when is first name ever in lower case

But why did we need email and last name comparison?